lib/base/style/colors.dart

```diff
@@ -9,8 +9,30 @@ abstract class AppColor {
   static const Color white = Color(0xffFAFAFA);
   static const Color gray = Color(0xff767676);
   static const Color lightGray = Color(0xffC2C2C2);
-  
+
   static const Color slightlyHighlighted = Color(0xffFEF4D2);
   static const Color success = Color(0xff4CAF50);
   static const Color error = Color(0xffFF4D00);
+
+  /// Creates a swatch from a given Color.
+  /// https://medium.com/@filipvk/creating-a-custom-color-swatch-in-flutter-554bcdcb27f3
```

Comment:
It's cool with `DOC comments`! Keep them coming
However I think the link medium should just be a regular comment as it gives no value in autogenerated code documentation.